cs.py

```diff
@@ -82,8 +95,16 @@ def main():
         from modules import azureaudit
         azureaudit.azure_audit()
         log.info("completed azure audit")
-        exit(0)
+
+
+    if args.number and args.wipe == False:    
+        if(number < 0):
```

Comment:
We should be consistent either we should use args.number or number.
Also if number is less then 0 will we ever reach to this if condition

Reply:
if args.number > 0 and args.wipe == False: